spec/ReactTags.spec.js

```diff
@@ -371,6 +375,17 @@ describe('React Tags', () => {
 
       expect($$('.custom-tag').length).toEqual(2)
     })
+
+    it('can receive tags through paste, respecting delimiters', () => {
```

Comment:
with the regexp update, you'll definitely want to put up some tests against active chars, like `delimiterChars = [']','.*[']`. It's not too hard to hijack someone's React application with a user script, so making sure the code doesn't die when an intentionally bad regex is dropped in will be worth the effort.